plugins/netflow.yaml

```diff
@@ -0,0 +1,43 @@
+# Plugin Info
+version: 0.0.1
+title: Netflow
+description: Log parser for Netflow
+min_stanza_version: 1.1.0
+parameters:
+  - name: listen_address
+    label: Listen Address
+    description: The IP address to bind to
+    type: string
+    default: "0.0.0.0:2055"
+  - name: version
+    label: Netflow Version
+    description: The netflow version (`v5`, `v9 / IPFIX`)
+    type: enum
+    valid_values:
+      - v5
+      - ipfix
+    required: true
+
+# Set Defaults
+# {{$listen_address := default "0.0.0.0" .listen_address}}
+# {{$port := default 2055 .port}}
```

Comment:
Since you removed port in a previous commit, the port here should be moved into listen_address here too, right?

Reply:
Good catch, resolved.